.gitignore

```diff
@@ -8,6 +8,8 @@ tmp.discovery
 **/node_modules
 web/web
 web/app/dist
+web/app/js/locales/_build
```

Comment:
From [docs](https://lingui.js.org/guides/excluding-build-files): 
`extract` command creates temporary message catalogs per each source file. Also,`compile` command generates compiled message catalogs from source ones. *All these files can be safely ignored from VCS and linters.*